src/app.rs

```diff
@@ -46,8 +46,8 @@ const DEFAULT_ROUTE: Route = Route {
 
 #[derive(Clone)]
 pub struct ScrollableResultPages<T> {
-    index: usize,
-    pages: Vec<T>,
+    pub index: usize,
+    pub pages: Vec<T>,
```

Comment:
Does this need to be public? The `impl` below had methods for fetching the data out of `pages`